OpenUpSA/wazimap-ng-ui
# Tooltip changes

## Description
modifying the tooltip depending on the choropleth method

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-493

## How is it tested automatically?
added `__tests__/gui/popup.feature`

## How should a reviewer test it locally
* select an indicator which is `subindicator` choropleth method and confirm that the tooltip and the mapchip are modified accordingly
* select an indicator which is `sibling` choropleth method and confirm that the tooltip and the mapchip are modified accordingly
* select an indicator which is `absolute` choropleth method and confirm that the tooltip and the mapchip are modified accordingly

## Screenshots
* subindicator choropleth method
![image](https://user-images.githubusercontent.com/53019884/196619128-8b0e1077-4793-4847-9384-c00431876873.png)

* sibling choropleth method
![image](https://user-images.githubusercontent.com/53019884/196619306-2c943d9c-8a1e-49cb-800d-5646bdb783c2.png)

* absolute choropleth method
![image](https://user-images.githubusercontent.com/53019884/196619391-2a1c2814-d79e-4d18-adc6-c40827cf1116.png)

## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
